test/processors/image_processor_test.rb

```diff
@@ -22,9 +22,9 @@ class ImageProcessorTest < ActiveSupport::TestCase
       }.with_indifferent_access
     }
 
-    it "should resize a local file" do
+    it 'should resize a local file' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.